### ISSUE TITLE
Workgroup Page and For-Profit Workgroups

### ISSUE THREAD

**Issue by Thunderosa** [monero-project/monero-site#1380]
As far as I know, Monero Space is the first private, for-profit company wanting to be listed as a Monero Workgroup, so this is a new decision for getmonero to make. As such, it would make sense to me that a general policy be established in case other corporations would like to be listed in the future.

MoneroOutreach will need to do likewise, although we've tended to take our guidance from getmonero on matters of which things to list, wallets etc. This is a very tricky decision, we can imagine all sorts of future scenarios/actors in this permissionless space; I'm not exactly sure where the right balance is.

So that's in general.

More specifically this is a company that was started by Justin (and NeedsMoney?) after trying to turn Monero Community into an LLC and abruptly quitting when the community resisted. This is a continuation of that and putting it on the official Monero website is an endorsement of that.

**Comment by erciccione** [comment]
As mentioned in [my reddit comment](https://www.reddit.com/r/Monero/comments/kd4t7j/the_old_team_page_on_getmoneroorg_has_been/gfxh8m1/), i've been thinking about adding a disclaimer for the monero-space workgroup, but at the end i avoided it.

It's true that this sets a precedent and personally i agree would be better put a disclaimer if the workgroup is managed by a corporation, especially since monero-space hasn't publicly declared that the workgroup is managed by a legal entity based in the US and the information is buried in the terms of service of their services.[1]

Not informing the users, but instead directing them to long ToS that nobody reads is a non-transparent action that doesn't inspire trust (mega corporations do this daily to hide partisan ToS). Monero-space decided to not inform their users about the legal status of the tools they are providing, that's their choice, but if we list the workgroup on getmonero, i agree we should make sure our users are fully aware.

In definitive, the structure of this particular workgroup diverge from all the other workgroups. In order to safeguard the privacy of the users, i too think a disclaimer is necessary. Something like:

*Warning: unlike the other workgroups listed on this page, this workgroup is a legal entity and the tools it provides are owned by an LLC company based in the US.*

> MoneroOutreach will need to do likewise

@Thunderosa is outreach considering becoming a company as well?


*[1] After the initial pushbacks, the leaders of monero-space declared they had to think about what to proceed forward with the LLC and that they would give an update on the decision. Instead of giving an update on their intentions, they simply created the LLC and launched their forum, without publicly disclosing or warning the users about the fact that the forum is managed by a board of a corporation based in the US. After asking @SamsungGalaxyPlayer why this wasn't shared publicly as they said they would have done, i got answered that the details about monero-space being owned by a corporation are in the Terms of Service of the forum itself.*

**Comment by Thunderosa** [comment]
No, we are absolutely not considering becoming a company. Sorry about that, I was meaning that like getmonero we'll also need to arrive at a policy for listing workgroups on our site.

MO has not had that discussion yet, I'd expect that if we decided to list a company as a Monero Workgroup we'd probably create a separate section for those so it's clear and list the full business name with LLC or Inc. etc. I don't think it's too outrageous to also list the jurisdiction and the URL and/or absence of their warrant canary; that is relevant information to the visitor and promotes basic transparency. Especially when a company isn't being transparent.

**Comment by SamsungGalaxyPlayer** [comment]
In my opinion (as an organizer of Monero Space), this removal would do far more harm than good. Monero Space functions just like any other workgroup and functionally meets any reasonable definition of a Monero workgroup. For example, we host monthly Monero Meet gatherings, conduct various community interviews, and have an active discussion in #monero-space. Though new, Monero Space is already an important part of the Monero community.

We could have a similar discussion for the wallet downloads page. That page links to open source, trusted wallets from companies such as MyMonero, Cake Wallet, and Edge. People are largely fine with this because they are good, open source wallets with trusted maintainers. The fact that they are maintained by a company is largely irrelevant.

I personally see the fact that a workgroup decided to form an entity to handle its finances and the like as irrelevant to it being a worthy enough workgroup to be listed on the getmonero.org website.

**Comment by erciccione** [comment]
> this removal would do far more harm than good

@SamsungGalaxyPlayer this discussion is about adding a warning on the square of Monero Space, nobody is suggesting a removal.

> We could have a similar discussion for the wallet downloads page

I don't see how the two are related. The section you are mentioning lists *only* external wallets/projects, that's why we have a disclaimer to make people do their own researches before using the software listed there. Doesn't matter if they are companies or not, because we have the disclaimer. The "Workgroup" page hosts a list of Monero workgroups, none of which, except one, is managed by a company. Putting a disclaimer here makes sense, since there is one single exception and since it's a significant one, people should be informed.

**Comment by SamsungGalaxyPlayer** [comment]
I don't see the difference when linking to external resources/efforts. If you want to have a disclaimer that any workgroup not run by Core is run by an external, possibly-malicious entity, then advocate for that. But since the Monero community is purposefully organized into such self-organized groups, I think a disclaimer is far more likely to confuse people and put them off from contributing than anything else. Setting a disclaimer for companies only is extremely arbitrary; the same logic applies more broadly to anything non-Core.

**Comment by erciccione** [comment]
I think it's useful and important to make people know that unlike the rest of the workgroups listed on that page, this particular one differs in the fact that it's managed by a US-based company (and all the tools are owned by this company as well). This has privacy-specific implications that a visitor/potential contributor needs to be aware of, especially since the only way to find out about the LLC is reading the terms of services. That's my opinion, i'm not going to jump into another endless debate about Monero Space.

**Comment by lh1008** [comment]
I think the disclaimer is necessary. We need to protect the real minority, the user, call the user, a company, a corporation, an individual, or a person.

**Comment by lh1008** [comment]
So the disclaimer should say who you're dealing with. A company/corporation/enterprise or a group of individuals not grouped together into a legal representation.

**Comment by xmrhaelan** [comment]
I think a disclaimer makes sense given everything that’s been said here, but it would be important to ensure the wording used is neutral as possible to avoid anyone passing undue judgement. I agree with Justin that not having Monero Space listed would be a bad thing. We should also recognize they are active in the community and have put a lot of their own time and resources into helping build what we have today. 

I think the question should be, if any other company that didn’t have people with such a long history in the community wanted to be listed, would we put a disclaimer?

**Comment by SamsungGalaxyPlayer** [comment]
My position remains that the distinction discussed here is completely arbitrary. If you are worried about a workgroup being run by bad actors, then remove them from the page. Any distinction has no relevance to a user who is viewing the page unless we want them to be the target of the fear-mongering derived from a complete misunderstanding of how organizations work.

**Comment by erciccione** [comment]
> I agree with Justin that not having Monero Space listed would be a bad thing.

@xmrhaelan Again: nowhere in this issue somebody is suggesting to remove monero-space from the list. Let's make that clear :)

> Any distinction has no relevance to a user who is viewing the page

@SamsungGalaxyPlayer this is false and misleading. We are not talking about people "viewing" pages, but subscribing to services that are forced to strictly follow US legislation, which includes retaining logs and making them available to police forces. The US is one of the most privacy-invasive countries in the world with very limited protections for its citizens (compared for example with the EU), even privacytools usually don't list tools/services developed by US-based companies, and if they do list them, they put a disclaimer like the one we are suggesting to add. In your opinion the distinction is useless? Ok. I think it's a very important one, especially considering the non-transparent behaviour of the creators of the workgroup.

> a complete misunderstanding of how organizations work.

Please consider that you are not the only person being aware of how organizations work. Just on this thread you have plenty of people disagreeing with you.

**Comment by SamsungGalaxyPlayer** [comment]
You keep moving the bar. Why do we allow wallets maintained by companies in the US? Surely by your logic they could be forced to collect information. Plus we still link to other services such as GitHub, Reddit, and Twitter which are US based and run by a company in the same workgroups page. You are imposing a higher standard on MS.

Even if you argue that GitHub is a necessary evil, that doesn't explain Twitter and Reddit. And I think most people would still rather link to Twitter and Reddit than not.

**Comment by Thunderosa** [comment]
Some wallet companies are listed, not all. All wallet companies are being listed in their own section and/or with a disclaimer.

So, it sounds like we're all in agreement that we should list workgroups the same way we've always listed wallets. 

If Github, Reddit and Twitter ask to be on the workgroup page, we can handle it the same way.

**Comment by SamsungGalaxyPlayer** [comment]
Github, Twitter, and Reddit links are on the workgroup page currently, and I think adding disclaimers that these services operate in the US would be pointless. All a user is going to do is get confused, and we don't want to be putting people off from joining various areas of the community.

We already describe workgroups at the top of the page in a way that describes them as self-organizing:

> Workgroups are formed by community members who join forces to achieve a common goal. They have different structures and goals.

Making this wording messy just makes things worse for everyone.

**Comment by lh1008** [comment]
Justin you're advocating for your personal interests and it's okay, but you're forgetting about the most important actor in Monero, and it's the users. We are advocating for the interests of the them, the user. I never liked listing any type of company to the site but you all pushed it anyway. 
Why I don't like listing companies of any type? Because we forget about building money and instead we build personal interests. So, like we already decided to push and list companies on the site we have to advocate for the liability of its users and Monero itself. Your acitons or a company's action don't represent the Monero community so whatever you do is your own liability. Not holding your own liability is avoiding your responsibility, so why did the LLC was created at first?

**Comment by lh1008** [comment]
So this is not a talk about Monero Space, it's a talk about Monero users. Users could be, companies, individuals, corporations, natural persons, juridical persons, etc. 
Adding the disclaimer is a social responsibility to make aware users about their own responsibility/liability to work with community members.

**Comment by rbrunner7** [comment]
How about something like a compromise?

On the one hand I think using the word "Warning" is too strong. If we start to put warnings in all places where we have "US inside" somehow that quickly escalates and does not make sense anymore.

On the other hand it currently is the only workgroup that is a legal entity, and it is a legal entity in the US, with that country having a somewhat special state in this world, so I find that noteworthy in the frame of Monero, and worthy of mentioning on the page in question. No warning, just a mention, as part of the workgroup description:

> Monero Space is an active, project-focused Monero workgroup that provides services to the Monero community. It hosts popular resources and organizes events such as Monero Meet. It is incorporated in the US as an LLC company.

**Comment by lh1008** [comment]
Yeah, @rbrunner7 with a disclaimer will be enough. Disclaimers are not only warnings but to specify obligations, liabilities, responsibilities, delimit rights, etc. In any case a user relates also with an individual they will have to acknowledge these liabilities too.

**Comment by erciccione** [comment]
Probably the word "warning" was a mistake. I think @rbrunner7's suggestion is very good.

Added with #1383

**Comment by ProkhorZ** [comment]
I wouldn't call it a disclaimer. Just call it a company when it's a company.

**Comment by binaryFate** [comment]
First a meta point: it is super important that key historical contributors are able to interact with each other in good faith. We need to be excellent to one another, even in disagreement, if we want to produce something excellent together. You'll forgive me the patronizing tone, but I find these discussions sometimes unnecessarily painful not for what is said but for how it is said.


Now on this particular issue (this is my personal opinion btw -- I don't know everyone's opinion on this within core team). I think it is a mistake to try to discuss and agree on a general policy aiming to answer a-priori what we would do in the future if another situation happens with another company. Especially as the topic is clearly difficult and divisive, we should favor pragmatism and focus on the particular situation happening today. We can debate about different future situations when they actually happen.

Because I know the important contributions of @SamsungGalaxyPlayer and needmoney90 since many years and believe (until witnessing something to the contrary) in their genuine motivations, they are not "any company" in my books.
I don't believe the privacy concerns are important enough in practice to add a disclaimer in this particular situation, on this particular workgroup page. I don't see anyone going like "I'll go and participate in this workgroup" and giving a lot of their private info straight away during discussions within the workgroup, to realize later "oh no it's a US company what did I say". Again I would favor pragmatism here, rather than fight each other on abstract principles.

We always run the danger of being seen as overly centralized when the website is acting as a gate keeper -- there is no way around it. Hence, at least on the specific page that is about self-organized groups, it would be healthy to allow some leeway to workgroups (from known contributors!) to be listed as they wish. Until any harm is done or likely, of course.

Finally, some arguments (and emotions I guess) in this thread are coming from previous heated discussions or referring to them. I think this is not productive, and I really wish we'd stop frequently bringing back an episode that has been unnecessarily divisive in the community.

**Comment by erciccione** [comment]
@binaryFate 

> Because I know the important contributions of @SamsungGalaxyPlayer and needmoney90 since many years and believe (until witnessing something to the contrary) in their genuine motivations, they are not "any company" in my books.

This is an extremely dangerous way of thinking IMO. First, they are a company, doesn't matter our personal opinions. Second, you may trust needomoney and sgp, but the board of directors include a total of 5 (?) people and the members could change any time. Will we constantly check the board members to make sure we trust them all?

> to realize later "oh no it's a US company what did I say"

This actually already happened when i made the post on reddit warning people about the legal status of Monero Space. Even if the post was up for only few hours, at least one person requested MS to have their data and account removed from their platforms and another one thanked me publicly on IRC saying that they were not aware of the situation and they didn't feel comfortable subscribing anymore.

I really don't want to spend too many words on this conversation, i think i spent enough. I disagree with most of binaryfate's points, but what i want to point out is:

The PR is not about adding a warning or a disclaimer. It's simply a *clarification* and literally everybody on this issue except for the creator of the LLC itself agree that would be good to have one. If the core team decides that even a clarification is too much, it's their decision and will obviously take their responsabilities about it. Personally, i think it would be a bad mistake, especially since the main reason for not adding a clarification doesn't seem to be based on anything more than the trust on 2 of the funding members of MS.

**Comment by fluffypony** [comment]
@erciccione I don't believe binaryFate was trying to say "trust this company" now and forever, he was merely pointing out that we cannot paint everyone with the same brush, especially when it involves long-time contributors that have, like you, given a great deal of their time, money, and effort to advance Monero.

In addition, I feel like much of the "fight" here is because some in the community have a certain ethos, and believe that every Monero contributor should have that same ethos. It's simply not possible for everyone in the Monero ecosystem to be anti-authoritarian, or anti-government, or anarcho-libertarian, or whatever other ideology they personally feel everyone should have. Monero already consists of many people with differing views. Some think it's ok to pay taxes. Some refuse to pay taxes as a protest against them. Some think it's ok to wear a mask when the government tells them to. Some refuse to wear a mask even if it has health benefits for their friends and family.

Ultimately, you or I or binaryFate or thankful_for_today may personally disagree with the way someone is choosing to contribute to the Monero ecosystem, but that doesn't give us free reign to be aggressive, belligerent, demeaning, or argumentative. Monero is going to continue to attract all sorts of people, and we should embrace their differences whilst educating them on using Monero as a tool. Let's not veer into territory where Monero is like a religion or a cult or a movement where everyone has to dress and act and think the same.

Last thing I'll add: when the Monero Outreach workgroup started, I was *very* put off by it. I have outspokenly stated on more than one occasion that Monero doesn't need a marketing department, and Outreach felt like a marketing department. Then they put out their first press release, and (if you'll excuse me saying so) it was garbage, confusing reporters at best. But I did not say much on the matter publicly. I side-channeled Outreach contributors now and then when I saw something that worried me, but I largely left them on their own to figure out how they felt Monero could best benefit from their efforts. Today, I am extremely proud of what they've accomplished, and how well organised and "professional" they've become. They have heard taken notice of the subtle nudges from the community, and continued to deliver a steady drumbeat of improvement, and a great deal of excellent content. I would really have hated it if I had expressed my initial concerns publicly, as it would not have changed anything, and might only have made frustrated them and caused them to stop contributing to Monero.

**Comment by rehrar** [comment]
**TL;DR I think a reasonable compromise is that Monero Space update their website and have a page disclosing what they are. As far as I'm concerned, Monero Space is a workgroup, and that workgroup is funded by an LLC. So let the workgroup stay on the workgroup page as a workgroup without disclaimer, and when people do digging to say where they get their funding, they can go to their website and see that they are funded by an LLC.**

I think a lot of attention is being given to this because of the 'precedent' it would set. Binaryfate alluded to the fact that we should really look at these things on a case-by-case basis rather than making overarching blanket rules that will apply to everyone at all times on these matters.

Yes, it's suckier, because case-by-case means exactly that. Somebody has to look at each individual case as opposed to just applying an even rule to everyone, but in a project like Monero that allows anonymity and pseudonymity, we're already doing that with how we handle reputation and trust.

This said, I really do understand that 'for profit' concern. Just because the current board doesn't care for profit, doesn't mean that will always be the case. And if we don't keep up with all future board changes, it might switch dramatically at the drop of a hat and our change to the website would be too little too late for some damage caused to some individuals. In this sense, I'm almost tempted to support putting the clarification. 

How this whole Monero Space project got started was tragic, and honestly, more than a little stupid. In the process though, I've seen sgp relent to the a vocal portion of the community again and again and again, even at great personal costs to projects he fronted and sustained through the entirety of their existence (i.e. coffee chat and the Monero Community youtube channel).

Why am I rambling about all of this? Because I think this clarification is unnecessary, and honestly, another slap in the face to sgp and needmoney90, and I don't think Monero Space being an LLC will make much of a difference to most. 

That said, I think a reasonable compromise would be to have Monero Space update their website to have a page where they clearly define what they are. Anyone who wants to do their due diligence before using any services will be able to find this information and make their choices from there.

The rest of the people don't care. If people think that literally everything Monero related is a community-driven thing with no businesses behind them, that's kind of on them. Bitcoin isn't like this. MyMonero has existed in the space for a long long time.

As far as I'm concerned, Monero Space is a workgroup, and that workgroup is funded by an LLC. So let the workgroup stay on the workgroup page as a workgroup without disclaimer, and when people do digging to say where they get their funding, they can go to their website and see that they are funded by an LLC.

**Comment by ProkhorZ** [comment]
@rehrar Nothing in your rant is a valid argument for not calling a company a company. It's not just about funding. It's not just about a precedent. **We are not Ripple**, we call things what they are. It's not presented as a big deal in #1383, and users who don't care can ignore that 12-word sentence. You know that many ignore much stronger warnings, like the advice to verify the hashes of downloads.

**Comment by umma08** [comment]
So far, I have seen these arguments put forward for not having a disclaimer:

1. The directors are known, trusted, and have committed time and effort to Monero.
2. The LLC helps to account for the tax bookkeeping of the directors (which in turn aids the supply of services for the community). 
3. That users don't care that a workgroup is a "for profit entity".
4. That a disclaimer will "put people off unnecessarily" (but it's not entirely clear why this is the case). 

The arguments for the disclaimer are

1. Privacy concerns over the US jurisdiction and its legal obligations.
2. The seemingly antithetical ideological message of having a "for profit entity" as a workgroup on an open source project. 
3. The somewhat chequered history of the workgroup itself.
4. The precedent this may set moving forward. 

A compromise may be to have a disclaimer on the workgroups that's aren't LLCs stating that they are "loose quorums of individuals working towards common goals and not formed as any legal entity or bound by any jurisdictional legal framework"

**Comment by fluffypony** [comment]
Just to add to my previous comment, I hold no strong opinion on whether or not there should be disclaimers, but I *am* advocating for people to act honourably and kindly to each other.

**Comment by erciccione** [comment]
> I feel like much of the "fight" here is because some in the community have a certain ethos, and believe that every Monero contributor should have that same ethos

I don't think so, but it's probably heated because some dislike MS and deem them unethical (that's the case for me, at least). Personally, i have enough experience with collective and orizzontal organizations that i know what to expect from a large community. I think the discussion here is swifting to an abstract argument about monero ethos and what we would like the community to be, when it's not actually the case, this is a strictly privacy-related matter.

Sure, i personally dislike Monero Space and the company like mentality they adopted from the beginning and i do think they are out of place, but as you may have noticed, i was never against listing them on the workgroups page. The fact that i dislike them sparked my "beligerant" tone, but the real thing to discuss here is simple:

Monero space is a workgroup managed by a US-based LLC. This 

1. has *practical* privacy implication for potential users of their services
2. is not the case for all the other workgroups listed

The proposed PR aims to clarify the legal status of the MS workgroup. We had the same type of conversation when we decided to list only open source wallets in the "downloads" page That's it, that's what we have to discuss.

The only concrete point for not adding the clarification seem to be focused about the fact the 2 of the funding members are trusted members of the community. If that's the metric, then there is really nothing i can add.

So please, comment about the merits of the suggested change, not abstract matters.

**Comment by binaryFate** [comment]
I don't believe that readers are approaching the workgroup page and then potentialy getting involved, with expectations that all people and infrastructure will preserve their privacy. I do not believe this is a realistic baseline position from which we must try to protect them.
For ex. all discussions taking place on IRC are public (and probably monitored in many channels), but we do not try to add a disclaimer that this is the case.

I see downsides in confusing readers, and perhaps most importantly I see downsides in being overdoing gatekeepers in general, which is actually making the project structure and community less healthy and less resilient.

**Comment by erciccione** [comment]
> I see downsides in confusing readers, and perhaps most importantly I see downsides in being overdoing gatekeepers in general

I struggle to understand how the phrase *"It is incorporated in the US as an LLC (Limited Liability Company)"* is gatekeeping or confusing. Anyway, at the moment the consensus (between here and #1383) is clear: 

7 people are for adding *'[...]. It's incomporated in the US as a LLC (Limited Liability Company)'*
3 people are for adding nothing (including 1 core team member)
2 people proposed alternatives (adding a disclaimer on the MS page or on the workgroup page)

Now it's up to core to take a decision. I think the discussion is too focused on either the contributors involved or perceived ideological matters, so personally, i don't have anything more to say.

**Comment by binaryFate** [comment]
Fwiw I think it would be nice if MS made it clear enough on their website (aka @rehrar's suggestion), in the interest of appeasing concerns that are understandable. Though I wouldn't make the workgroup page (lack of) wording necessarily conditional on that.

**Comment by SamsungGalaxyPlayer** [comment]
Just to reiterate a few points:

1. Saying "maybe the company will seek to earn profit in the future" is nonspecific to a company. What if Monero Outreach stated trying to make money by selling ads on their site? Workgroups change over time and the site can be updated if a workgroup starts becoming abusive. Suggesting this is specific to companies only is incorrect.

2. If people truly are scared about the US, it makes no sense to only include a disclaimer for Monero Space while ignoring the Monero Community Workgroup (organized by a US individual). Further, other workgroups may be partially organized by individuals in the US too.

These arbitrary high bars seem to only apply to Monero Space. The fact that we're having this discussion here shows that MS is willing to be transparent about what we do and why. Any workgroup could be run by a for-profit company or NSA employee *right now* and no one would know.

**Comment by rbrunner7** [comment]
@SamsungGalaxyPlayer : Some comments about the proposal to state your form of organization somewhere on the Monero Space website? I could imagine that as a third link beside "email" and "github" named "legal form" or similar that leads to a page that basically contains the sentence that this PR wants to add directly to the workgroup page?

**Comment by SamsungGalaxyPlayer** [comment]
I'm happy to include a link to our corporate docs.

https://wyobiz.wyo.gov/Business/FilingDetails.aspx?eFNum=148182127005251116092123125219038058020252005207

**Comment by erciccione** [comment]
> If people truly are scared about the US, it makes no sense to only include a disclaimer for Monero Space while ignoring the Monero Community Workgroup (organized by a US individual). Further, other workgroups may be partially organized by individuals in the US too.

@SamsungGalaxyPlayer  as pointed out in this issue many times, there is a huge difference in legal terms (responsabilities, risks, legal obbligations, etc) between a natural entity (physical person) and a legal entity (company) owning services or tools. You keep assuming that they are the same thing, but they are absolutely not, your comparison is meaningless. The difference is fundamental in Roman law. Please inform yourself.

> If people truly are scared about the US

Shouldn't they be scared of one of the most authoritarian and privacy-invasive countries in the world with a long record of human rights violations?

**Comment by Thunderosa** [comment]
It seems like we got a little sidetracked about if this is a warning, a disclaimer, a label or just information. I suggested forming a policy to avoid singling out, I think it's better to solve it. I hope we can all agree that transparency is important. Personally I feel transparency has been missing from this corporation since the beginning and that's why I've been more on guard than I would be otherwise. Asking for trust only makes me trust less.

The last things I'll say on this is:

- I'd prefer Justin not try to hide his status.
- I'd prefer Justin not demand that other workgroups help hide his status.
- I'm content to leave this to Core when it comes to the official site.

**Comment by SamsungGalaxyPlayer** [comment]
fwiw, I've never tried to hide anything. And I've added a link to our org docs on the website. https://monero.space/

**Comment by erciccione** [comment]
Personally, i don't doubt the good faith. The link is nice to have, but doesn't really solve the problem unless is also added on the pages we link to from getmonero: the forum. I know it's already in the terms of service, but nobody reads those.

**Comment by Thunderosa** [comment]
The filing papers don't list partners or describe governance.

**Comment by luigi1111** [comment]
Thanks all for your thoughts.

We are going to keep the self-organized, workgroup-provided descriptions for now. If bad faith is shown or it otherwise becomes untenable in the future, we will revisit the issue.

**Comment by erciccione** [comment]
@luigi1111 Could you expand about the reasoning behind this choice? [The vast majority of the people](https://github.com/monero-project/monero-site/issues/1380#issuecomment-751346757) expressed their will to clarify the legal status of Monero Space. The core team is deciding to go against the broad consensus reached by the community and it's not considering any of the alternatives offered, i think would be important to have an explanation of the reasoning behind the choice.

**Comment by endorxmr** [comment]
Personally I am strongly in favor of adding the LLC disclaimer on the wordgroups page. I for one would like to stay away from US-based LLCs as much as possible, and the fact that there's so much resistance about adding such a distinction on the offical Monero website raises a lot of red flags for me - compounding those already raised by the way this specific workgroup was started. So as far as I'm concerned, the fact that this minor (and relatively neutral) addition to their description is such a big deal *does* raise bad faith suspicions for me.

To me, this argument has parallels to the situation of Facebook vs Apple: if you're scared that people left you if they really knew what you were doing, then your business has no reason to exist. If Monero Space is so afraid of users leaving/not joining after reading about their US LLC status, then perhaps that group shouldn't exist in the first place. But that's a different topic I guess.

To those arguing that _most_ users won't care: that's not the point. Most people don't care about privacy either, yet companies have to include privacy policies. Monero's goal is to protect *everyone*, not just "some", or "the most". _Some_ users _will_ care about Monero Space's LLC status, and adding a few words to increase awareness _will_ help protect them, since the only other place to find this information is one link at the bottom of M.S.'s official website.

The fact that we already have to interact with so many other US-based does not mean that we should give Monero Space a "free pass" just because we trust them.

Monero Space LLC is a legal entity, and there's nothing wrong in stating that openly. Monero is about trustlessness and treating everyone fairly: therefore, all workgroups backed by financial and/or legal entities should be openly defined as such.

This should have never been controversial in the first place, for anyone. Transparency in public entities is objectively good. Fighting against it makes you suspect.

**Comment by erciccione** [comment]
> he fact that there's so much resistance about adding such a distinction on the offical Monero website raises a lot of red flags for me

This is the problem. I really don't understand how adding to the description of a workgroup `It's incorporated in the US as a LLC` is something controversial and to fight fiercely against. So much that the Core team felt they needed to intervene and revert the consensus reached by the the people involved in the conversation.

I hope to read a clarification from @luigi1111 or others soon, also because part of the reasoning is "We are going to keep the self-organized, **workgroup-provided** descriptions for now." (enphasis mine), which was never the case. The descriptions have been provided by me and edited by members of the workgroups in a couple of cases, so that's not really a point.

**Comment by rehrar** [comment]
> The vast majority of the people expressed their will to clarify the legal status of Monero Space.

I don't think that's the case here at all. Between the two people for an alternative (mine off site and rbrunner on site), and selsta's view in the PR that it was unnecessary, we're looking at 5 or 6 against and 7 for. This is nowhere close to the 'vast majority of people'. It seems pretty evenly split, and at the very least, contentious. 

It is not 'the will of the community' to clarify this. It is the will of some. Under contentious circumstances, it is not uncommon for Core to make a decision on the platforms that they maintain and steward. They have done so here (partially) under reasoning of assumption of good faith (which can be reverted) and that this is unnecessarily divisive at present time (further expansion can be given by a core team member).

**Comment by xmrhaelan** [comment]
Those numbers are rather small and probably do not account for the many people who were arguing this (on either side) on IRC a few weeks ago.

A subtle label does not seem like too much to ask. Transparency is a good thing in this regard for reasons already stated by others.

**Comment by erciccione** [comment]
@rehrar 

> I don't think that's the case here at all. Between the two people for an alternative (mine off site and rbrunner on site), and selsta's view in the PR that it was unnecessary, we're looking at 5 or 6 against and 7 for.

I have no idea where those numbers come from. If you count the public opinions on this issues and on the PR, it's clear that about double the amount of people agree a change is necessary (8 against 4, without even counting who offered alternatives). Let's try to be intellectually honest.

> this is unnecessarily divisive at present time

I and others, really struggle to see the clarification proposed as "divisive". The only problem here seems to be that this change is completely refused by at least one board member of Monero Space (and that's a problem of it's own, as @endoxmr explained very well) and to avoid drama core decided to simply avoid the problem. As some have already explained, not adding such innocuous change rises red flags and i really hope the core team will provide us with an explanation.

**Comment by lh1008** [comment]
I still suggest a disclaimer.

**Comment by luigi1111** [comment]
> I hope to read a clarification from @luigi1111 or others soon, also because part of the reasoning is "We are going to keep the self-organized, **workgroup-provided** descriptions for now." (enphasis mine), which was never the case. The descriptions have been provided by me and edited by members of the workgroups in a couple of cases, so that's not really a point.

It is (at least part of) the point. If there's a conflict (which there very clearly is here), the workgroup has first priority, unless good enough reason is found. That's not the case here, unless you or someone else has a more convincing argument about material differences between US persons and US companies than I've seen so far.

**Comment by Thunderosa** [comment]
I disagree strongly, but said I'd be OK with Core's decision. I'll reopen this ticket the next time Justin divides the community with hubris - well, the next-next time.
